lib/wings/valkyrie/persister.rb

```diff
@@ -37,7 +37,7 @@ def save_all(resources:)
       # @param [Valkyrie::Resource] resource
       # @return [Valkyrie::Resource] the deleted resource
       def delete(resource:)
-        af_object = resource_factory.from_resource(resource: resource)
+        af_object = ActiveFedora::Base.find(resource.alternate_ids.first.to_s)
```

Comment:
Could this behavior be baked into `from_resource`?

It seems like it might be useful in other contexts as well, and the switch to a hard dependency on AF, as opposed to continuing to use the factory, feels like a regression.

Is there more to say about this?

Reply:
like by adding a flag on the factory method to just fetch it instead of converting it?

Reply:
@no-reply I don't really understand. there are ActiveFedora dependencies all throughout the persister and the query service. I think that is expected. The persister and query service have to interact directly with the data store.